### DISCUSSION TITLE
UiKit vs Zurb Foundation

### DISCUSSION THREAD

**Discussion by amulatri** [uikit/uikit#4476] [General]
I have been experimenting with Foundation but found documentation it is not so well organized making it difficult to understand and customise. Recently started exploring UiKit and found it is much better and has impressive components as well overall features. However I feel UiKit team's main focus is on Joomla and Wordpress themes. Therefore not comfortable yet on focussing on UiKit. Any ocmments?

**Comment by mattfletcher** [comment]
We use UIkit at my work for a number of large websites and heavy-duty web applications, as well as dozens of pet projects. None of them have anything to do with Wordpress or Joomla. It's a great general purpose CSS framework/component library and I'd absolutely recommend it.

**Reply by amulatri** [reply]
Thanks Matt for relevant feedback.
